draft-ietf-quic-http.md

```diff
@@ -139,8 +139,8 @@ Alternative Services; this process is described in greater detail in
 Within each stream, the basic unit of HTTP/3 communication is a frame
 ({{frames}}).  Each frame type serves a different purpose.  For example, HEADERS
 and DATA frames form the basis of HTTP requests and responses
-({{request-response}}).  Other frame types like SETTINGS, PRIORITY, and GOAWAY
-are used to manage the overall connection and relationships between streams.
+({{request-response}}).  Other frame types like SETTINGS and GOAWAY are used to
+manage the overall connection and relationships between streams.
```

Comment:
A minor contention: IMO SETTINGS or GOAWAY don't exemplify "relationships between streams". In fact, removing tree depencies helps makes request streams way more independent.

Reply:
Do you think that I should just strike that sentence?

Reply:
```suggestion
manage the overall connection.
```

yes, or do this.